packages/css/src/plugins/expand-shorthands/index.tsx

```diff
@@ -58,31 +58,36 @@ const shorthands: Record<string, ConversionFunction> = {
 /**
  * PostCSS plugin that expands shortform properties to their longform equivalents.
  */
-export const expandShorthands = plugin('expand-shorthands', () => {
+export const expandShorthands = (): Plugin => {
   const filter = new RegExp(Object.keys(shorthands).join('|'));
 
-  return (root) => {
-    root.walkDecls(filter, (decl) => {
-      const valueNode = parse(decl.value);
-      const expand = shorthands[decl.prop];
+  return {
+    postcssPlugin: 'expand-shorthands',
+    Once(root) {
```

Comment:
This can be replaced with:

```tsx
Declaration(declaration) {
  if (!shorthands[declaration.prop]) {
    return;
  }

  // remaining logic here...
}
```

or this, which should be the fastest:

```tsx
const onDeclaration = (declaration: Declaration) => {
  // remaining logic here...
};

// ...
Declaration: Object.keys(shorthands).reduce((keys, shorthand) => {
  keys[shorthand] = onDeclaration;
  return keys;
}, {})
```

Refer to https://github.com/postcss/postcss/blob/main/docs/guidelines/plugin.md#23-use-fast-nodes-scanning

Reply:
I think the faster method lacks the typescript support, have gone with the first suggestion which is working fine.

Reply:
Awesome! Though I don't see the first change here, did you forget to push it up?

Reply:
I sure did, all pushed now